src/unk_0206A8DC.c

```diff
@@ -103,19 +103,19 @@ BOOL sub_0206A9B4 (UnkStruct_020507E4 * param0)
     return sub_0206A8EC(param0, (2400 + 5));
 }
 
-void sub_0206A9C4 (UnkStruct_020507E4 * param0)
+void Events_SetVsSeekerUsedFlag (UnkStruct_020507E4 * param0)
 {
     sub_0206A8DC(param0, (2400 + 13));
     return;
 }
 
-void sub_0206A9D4 (UnkStruct_020507E4 * param0)
+void Events_ClearVsSeekerUsedFlag (UnkStruct_020507E4 * param0)
 {
     sub_0206A8E4(param0, (2400 + 13));
     return;
 }
 
-BOOL sub_0206A9E4 (UnkStruct_020507E4 * param0)
+BOOL Events_GetVsSeekerUsedFlag (UnkStruct_020507E4 * param0)
 {
     return sub_0206A8EC(param0, (2400 + 13));
 }
```

Comment:
Document this subroutine if you can do so.

Reply:
What would I even label this? It calls `VarsFlags_IsFlagOn`, directly forwarding both parameters

Reply:
Well that's a little annoying... Let's leave it for now. Maybe a name becomes more apparent with more documentation.